### PR TITLE
Feat: add support for checking intezer site availability.

### PR DESCRIPTION
**Pull request by guysl10**
Support this API:
`https://analyze.intezer.com/api/v2-0/is-available`

### CODE REVIEW

File: intezer_sdk/api.py [hunk 1]
```diff
@@ -219,6 +219,12 @@ def assert_any_on_premise(self):
         if self.on_premise_version:
             raise errors.UnsupportedOnPremiseVersionError('This endpoint is not available yet on on-premise')
 
+    @staticmethod
+    def is_intezer_site_available() -> bool:
+        response = requests.get('https://analyze.intezer.com/api/v2-0/is-available')
+        is_available = response.json().get('result', {}).get('Is available')
```
Comment (davidt99) [inline intezer_sdk/api.py:225]: Are you sure it's `Is available`, the docs says otherwise

Comment (davidt99) [inline intezer_sdk/api.py:225]: I think json will fail if the response didn't returned json formatted, I suggest we check the status code before

Comment (guysl10) [reply intezer_sdk/api.py:225]: You are right, I got confuse with Splunk soar:
image
image

**Review by davidt99 — APPROVED** [review]

File: intezer_sdk/api.py [hunk 2]
```diff
@@ -219,6 +219,14 @@ def assert_any_on_premise(self):
         if self.on_premise_version:
             raise errors.UnsupportedOnPremiseVersionError('This endpoint is not available yet on on-premise')
 
+    @staticmethod
+    def is_intezer_site_available() -> bool:
+        response = requests.get('https://analyze.intezer.com/api/v2-0/is-available')
+        if response.status_code == HTTPStatus.OK:
+            is_available = response.json().get('result').get('is_available')
```
Comment (matany90) [inline intezer_sdk/api.py:226]: add {} fallback to the result

**Review by matany90 — APPROVED** [review]

**Pull request merged**